frontend/views/components/Chatroom/ChatMain.vue

```diff
@@ -302,37 +161,8 @@ export default {
     startedUnreadIndex () {
       return this.details.conversation.findIndex(message => message.unread === true)
     },
-    getMessageAt () {
-      let isCurrentUser
-
-      return this.details.conversation.map((message, index) => {
-        isCurrentUser = this.currentUserAttr.id === message.from
-
-        if (message.from === messageTypes.INTERACTIVE) {
-          return {
-            is: MessageInteractive,
-            id: message.id
-          }
-        }
-        if (message.from === messageTypes.NOTIFICATION) {
-          return {
-            is: MessageNotification,
-            text: message.text
-          }
-        }
-
-        return {
-          is: Message,
-          text: message.text,
-          who: this.who(isCurrentUser, message.from),
-          avatar: this.avatar(isCurrentUser, message.from),
-          variant: this.variant(isCurrentUser),
-          hideWho: this.shouldHideWho(index),
-          isSameSender: this.isSameSender(index)
-        }
-      })
-    },
     getPendingAt () {
+      console.log('PLOP')
```

Comment:
Why was `getMessageAt` deleted, and what is this `'PLOP'`?

Reply:
getMessageAt was deleted because the code was just obfuscating from the markup what kind of messages was used for no reason. It was also creating an extra loop for no reason and was displaying twice message notifications.
PLOP is just a reminder to myself to do something (normally before commit). In this case it was to come back here and add the pending messages in the markup.

Reply:
Ah, I see, nice call! 👍

Reply:
(This is code I hadn't reviewed in depth because I figured it would all be rewritten, so basically anything related to the chat, feel free to improve things! The logic for it will likely need to be rewritten at some point.)